tests/e2e/helpers/common.ts

```diff
@@ -263,6 +263,26 @@ export class Common {
         }
     }
 
+    /**
+     * Delete folder
+     * @param filePath Path to file
+     */
+    static async deleteFolderIfExists(filePath: string): Promise<void> {
+      if (fs.existsSync(filePath)) {
+            fs.rm(filePath, {
+                recursive: true,
+                force: true
+            }, (error) => {
+                if (error) {
+                    console.log(error);
+                }
+                else {
+                    console.log(`Directory Deleted: ${filePath}`);
+
+                }
+            });
+        }
+    }
```

Comment:
```suggestion
    static async deleteFolderIfExists(filePath: string): Promise<void> {
        try {
            await fs.rm(filePath, { recursive: true, force: true });
            console.log(`Directory Deleted: ${filePath}`);
        } catch (error) {
            console.error(`Failed to delete directory: ${filePath}`, error);
        }
    }
```

as per [documentation](https://nodejs.org/api/fs.html#fspromisesrmpath-options) `force: true` would prevent missing file exceptions (a.k.a. if file is there it will be deleted; if not - no error) so we can safely remove `fs.existsSync(filePath)`